app/views/log_entries/index.haml

```diff
@@ -19,11 +19,11 @@
           - if @current_user.can_modify? entry
             .edit-form
               = bootstrap_form_with namespace: entry.id, model: entry,
-                url: log_path(entry), local: true do |f|
+                url: log_entry_path(entry), local: true do |f|
```

Comment:
We can just remove the `url:`. If you give `form_with` a `model:`, it will infer the path.